src/controller/java/AndroidOperationalCredentialsIssuer.cpp

```diff
@@ -47,7 +47,8 @@ static CHIP_ERROR N2J_CSRInfo(JNIEnv * env, jbyteArray nonce, jbyteArray element
 
 static CHIP_ERROR N2J_AttestationInfo(JNIEnv * env, jbyteArray challenge, jbyteArray nonce, jbyteArray elements,
                                       jbyteArray elementsSignature, jbyteArray dac, jbyteArray pai, jbyteArray cd,
-                                      jbyteArray firmwareInfo, jobject & outAttestationInfo);
+                                      jbyteArray firmwareInfo, chip::VendorId vendorId, uint16_t productId,
```

Comment:
Use uint16_t for VendorID, not `chip::VendorId` which is a very partial enum, and is getting phased out.

Reply:
Modified. Thanks.